presto-main/src/main/java/com/facebook/presto/server/HttpRequestSessionContext.java

```diff
@@ -100,38 +97,26 @@ public HttpRequestSessionContext(HttpServletRequest servletRequest)
         clientInfo = servletRequest.getHeader(PRESTO_CLIENT_INFO);
         clientTags = parseClientTags(servletRequest);
 
-        // parse session properties
-        ImmutableMap.Builder<String, String> systemProperties = ImmutableMap.builder();
-        Map<String, Map<String, String>> catalogSessionProperties = new HashMap<>();
-        for (Entry<String, String> entry : parseSessionHeaders(servletRequest).entrySet()) {
-            String fullPropertyName = entry.getKey();
-            String propertyValue = entry.getValue();
-            List<String> nameParts = Splitter.on('.').splitToList(fullPropertyName);
+        // parse session properties, catalog session properties can not be validated until the transaction has stated, so we delay all property validation
```

Comment:
This seems like two separate sentences. It's also confusing since this code no longer has any references to catalog properties.